.github/workflows/push.yml

```diff
@@ -104,7 +104,7 @@ jobs:
                     -e KBC_URL \
                     -e KBC_TOKEN \
                     -e WORKSPACE_CONFIGURATION_ID \
-                    ${{env.APP_IMAGE}} composer ci
+                    app-tests
```

Comment:
nelíbí se mi že to testuješ oproti úplně jinému docker image než který je v produkci 🤔 
to bude potřeba vymyslet jinak

Reply:
Co takhle?